components/public-api-server/pkg/auth/personal_access_token.go

```diff
@@ -45,7 +45,7 @@ func (t *PersonalAccessToken) Value() string {
 }
 
 func (t *PersonalAccessToken) ValueHash() (string, error) {
-	const bcryptCost = 16
+	const bcryptCost = 10
```

Comment:
Should this value be configurable or are you happy to hard-code it?

Reply:
Happy to hard-code it. In a way, it should only be changed in rare scenarios and for those a PR is okay. It's made a constant just to make it more explicit, but also communciate what that value actually does